src/core/ext/filters/client_channel/lb_policy/xds/xds_cluster_resolver.cc

```diff
@@ -244,7 +250,8 @@ class XdsClusterResolverLb : public LoadBalancingPolicy {
       // in methods of this class rather than in lambdas to work around an MSVC
       // bug.
       void OnResourceChangedHelper(
-          std::shared_ptr<const XdsEndpointResource> update) {
+          std::shared_ptr<const XdsEndpointResource> update,
+          RefCountedPtr<XdsApi::ReadDelayHandle> /*read_delay_handle*/) {
```

Comment:
I think this needs to be passed through to `OnEndpointChanged()`.

Reply:
Actually, sorry, I think I steered you wrong on this -- the lambda won't go out of scope until after we return from `OnEndpointChanged()`, so there's not actually any reason why we need to pass the handle down there.

In fact, I don't even think the handle needs to be plumbed into any of these `*Helper()` methods.  I think it's sufficient to capture it in the lambda that runs in the `WorkSerializer`.